cmd/assetsvc/postgresql_utils.go

```diff
@@ -39,6 +39,16 @@ type postgresAssetManager struct {
 	dbutils.PostgresAssetManagerIface
 }
 
+type chartQuery struct {
+	namespace   string
+	chartName   string
+	version     string
+	appVersion  string
+	searchQuery string
+	repos       []string
+	categories  []string
+}
```

Comment:
I don't know if it is the best place to put this struct, especially due to this is not an exclusive model for postgresql , perhaps in utils.go?